src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -100,9 +105,7 @@ Bluetooth 4.0 or later.
 
 When a website requests access to nearby devices using
 `navigator.bluetooth.requestDevice`, Google Chrome will prompt user with a
-device chooser where he can pick one device or simply cancel the request. At
-the time of writing though, the device chooser hasn't been implemented yet in
-Chrome OS. Only the first device that matches filters will be returned.
+device chooser where he can pick one device or simply cancel the request.
```

Comment:
Damn! 
I'm trying to be careful with those but I've missed this one ;(

Thank you.